### ISSUE TITLE
Add labels for inputs

### ISSUE THREAD

**Issue by jpdevries** [jpdevries/eureka#56]
Add label for `#media-browser_0__filter-images`

![](http://j4p.us/032H0o3H2l0P/Screen%20Shot%202016-02-19%20at%202.06.35%20PM.png)

**Comment by jpdevries** [comment]
Should be fixed by breaking changes, VirtualDOM. Now testing with axe-core and more.
